src/devices/sound/es5506.cpp

```diff
@@ -159,7 +159,7 @@ es550x_device::es550x_device(const machine_config &mconfig, device_type type, co
 	, m_volume_shift(0)
 	, m_volume_acc_shift(0)
 	, m_current_page(0)
-	, m_active_voices(0)
+	, m_active_voices(0x1f)
```

Comment:
Why are the active voices on construction in this file changed, and why are a couple of C++ line comments changed to C-style comments?

Reply:
The active voices are set to 0x1f on reset, but they previously were set to 0 in the constructor. This causes an unnecessary and significant sample rate ping-pong at startup (32x difference), since the sample rate is dependent upon the number of voices. Initializing it to the same value here removes the sample rate change.

Reply:
No idea how those C++ comments got changed; that's weird.